components/brave_wallet/browser/keyring_service.cc

```diff
@@ -1208,8 +1290,8 @@ bool KeyringService::IsHardwareAccount(const std::string& account) const {
 }
 
 void KeyringService::Unlock(const std::string& password,
-                            UnlockCallback callback) {
-  if (!ResumeKeyring(kDefaultKeyringId, password)) {
+                            KeyringService::UnlockCallback callback) {
+  if (!ResumeKeyring(mojom::kDefaultKeyringId, password)) {
```

Comment:
shouldn't we also resume filecoin keyring?

Reply:
will add it when UI ready, it is not necessary now